__tests__/app.test.js

```diff
@@ -13,153 +13,320 @@ beforeEach(() => {
   return seed(data);
 });
 
-describe("app", () => {
-  describe("GET /api/categories", () => {
-    test("status:200 and response message", () => {
-      return request(app).get("/api/categories").expect(200);
-    });
-    describe("GET /api/categories", () => {
-      test("returns an array of objects with a slug and desc", () => {
-        return request(app)
-          .get("/api/categories")
-          .expect(200)
-          .then((res) => {
-            res.body.categories.forEach((category) => {
-              expect(typeof category).toBe("object");
-              expect(category.hasOwnProperty("description")).toBe(true);
-              expect(category.hasOwnProperty("slug")).toBe(true);
-            });
-            expect(res.body.categories.length).toBeGreaterThan(1);
-          });
+test("GET /api/categories : status:200 and response message", () => {
+  return request(app).get("/api/categories").expect(200);
+});
+
+test("GET /api/categories : returns an array of objects with a slug and desc", () => {
+  return request(app)
+    .get("/api/categories")
+    .expect(200)
+    .then((res) => {
+      res.body.categories.forEach((category) => {
+        expect(typeof category).toBe("object");
+        expect(category.hasOwnProperty("description")).toBe(true);
+        expect(category.hasOwnProperty("slug")).toBe(true);
       });
+      expect(res.body.categories.length).toBeGreaterThan(1);
     });
-  });
-  describe("GET /api/reviews", () => {
-    test("status:200 and response message", () => {
-      return request(app).get("/api/reviews").expect(200);
-    });
-    describe("GET /api/reviews", () => {
-      test("returns an array of objects with an owner, title, review_id, category, review_img_url, created_at, votes, designer", () => {
-        return request(app)
-          .get("/api/reviews")
-          .expect(200)
-          .then((res) => {
-            res.body.reviews.forEach((review) => {
-              expect(typeof review).toBe("object");
-              expect(review.hasOwnProperty("owner")).toBe(true);
-              expect(review.hasOwnProperty("title")).toBe(true);
-              expect(review.hasOwnProperty("review_id")).toBe(true);
-              expect(review.hasOwnProperty("category")).toBe(true);
-              expect(review.hasOwnProperty("review_img_url")).toBe(true);
-              expect(review.hasOwnProperty("created_at")).toBe(true);
-              expect(review.hasOwnProperty("votes")).toBe(true);
-              expect(review.hasOwnProperty("designer")).toBe(true);
-            });
-          });
+});
+test("GET /api/users : status:200 and response message", () => {
+  return request(app)
+  .get("/api/users")
+  .expect(200);
+});
```

Comment:
This test isn't needed as you'll expect 200 in the next test!

Reply:
Thanks, i have a few of these throughout, i will delete them all :)